service_container.rst

```diff
@@ -845,7 +860,7 @@ But, if you *do* need to make a service public, override the ``public`` setting:
             # ... same code as before
 
             # explicitly configure the service
-            App\Service\MessageGenerator:
+            Acme\PublicService:
```

Comment:
If this belongs to the previous example in this PR it should be MessageGenerator, shouldn’t it?

Reply:
It does not, the previous example makes use of a public twig service, here we define a specific public service, should I change something to make it more clear?

Reply:
On jo let’s keep it like this